CMakeLists.txt

```diff
@@ -368,12 +370,6 @@ if( CCACHE_FOUND )
   set_property( GLOBAL PROPERTY RULE_LAUNCH_LINK ccache )
 endif()
 
-
-if( VVENC_ENABLE_THIRDPARTY_JSON )
-  set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DVVENC_ENABLE_THIRDPARTY_JSON" )
-  set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DVVENC_ENABLE_THIRDPARTY_JSON" )
-endif()
```

Comment:
I'm starting to see why this was duplicated here.

Reply:
Good catch!